weaviate/backup/backup.py

```diff
@@ -24,6 +26,14 @@
 }
 
 
+class BackupCompressionLevel(str, Enum):
+    """Which compression level should be used to compress the backup."""
+
+    DEFAULT = "DefaultCompression"
+    BEST_SPEED = "BestSpeed"
+    BEST_QUALITY = "BestCompression"
```

Comment:
Should we use Literals? I know we have an Enum for the status but it feels like a mistake now.

Reply:
I think it's best to stick to the current convention. Having enums helps users that don't use type hinting and `mypy`, which I fear is quite a lot of the python community, so it has value there

Do you think the naming convention is okay?

Reply:
> I think it's best to stick to the current convention. Having enums helps users that don't use type hinting and mypy, which I fear is quite a lot of the python community, so it has value there

oki

> Do you think the naming convention is okay?

if these are the names of the options: sure :) What other naming convention are you thinking of?

Reply:
Just `BEST_QUALITY` instead of `BEST_COMPRESSION`. Actually, now that I'm looking at this again I think I prefer `BEST_COMPRESSION`. WDYT?

Reply:
yep, let's change to BEST_COMPRESSION